packages/terra-hookshot/src/HookshotContent.jsx

```diff
@@ -122,6 +135,24 @@ class HookshotContent extends React.Component {
     }
   }
 
+  enableContentResizeListener() {
+    if (!this.contentResizeListenerAdded) {
+      this.resizeObserver = new ResizeObserver((entries) => {
+        this.props.onContentResize(entries[0].contentRect);
+      });
+      this.resizeObserver.observe(this.contentNode);
+      this.contentResizeListenerAdded = true;
+    }
+  }
+
+  disableContentResizeListener() {
+    if (this.contentResizeListenerAdded) {
+      this.resizeObserver.disconnect(this.contentNode);
```

Comment:
To be safe we should disconnect the resize observer when the component unmounts

Reply:
Good catch, I had missed that one.